furnace.c

```diff
@@ -333,6 +378,15 @@ do_pwm_work(furnace_context_t *ctx)
   gpio_put(FURNACE_FIRE_PIN, ctx->pwm_current);
 }
 
+void
+do_pid_work(furnace_context_t *ctx, bool deadline_met){
```

Comment:
This project places the `{` on a new line. Please, format the code the same way it was before you  added your changes.